packages/express-backend/src/persistence/mongoose-connection.js

```diff
@@ -88,6 +88,30 @@ async function createItem(item, uid) {
   }
 }
 
+async function addItemToUserBought(item, uid) {
```

Comment:
It seems like this function creates a new item in the database and adds the newly created item to the item creator's `items_bought` list.

"Selling an item" is really just giving an existing item an id in the `buyer_id` field and updating the buyer's `bought_items` list to include the newly received item id.